pairtools/lib/stats.py

```diff
@@ -645,6 +651,30 @@ def reducer(accumulator, element):
                             {},
                         )
                         # sum_stat[k][dirs] = self._stat[filter][k][dirs] + other._stat[filter][k][dirs]
+                elif k == "chromsizes":
+                    if k in self._stat[filter] and k in other._stat[filter]:
+                        if self._stat[filter][k] == other._stat[filter][k]:
+                            sum_stat._stat[filter][k] = self._stat[filter][k]
+                        elif (
+                            len(self._stat[filter][k]) == 0
+                            or len(other._stat[filter][k]) == 0
+                        ):
+                            logger.warning(
+                                "One of the stats has no chromsizes recorded, using the one that is present"
```

Comment:
1. Will benefit from more clear reformulation: "One of the stats has no chromsizes recorded, will re-write the output with chromsizes present in other file."
2. If one of the files does not have "chromsizes" key at all, this warning won't show up, right? Might be inconsistent

Reply:
1 - ok!
2 - since "chromsizes" is now created when the object is created, it's always there, but might be empty if the file didn't have chromsizes